src/legacy-minter-filter-mapping.ts

```diff
@@ -337,6 +340,18 @@ export function loadOrCreateMinterFilter(
     coreContractAddress.toHexString(),
     timestamp
   );
+
+  // IMinterFilterV0 has no minterFilterType() function, however MinterFilterV1
+  // which implements IMinterFilterV0 does have a minterFilterType() function.
+  // We can use this to determine if the minter filter is a V0 or V1 minter filter.
+  const potentialV1MinterFilter = MinterFilterV1.bind(minterFilterAddress);
+  let minterTypeResult = potentialV1MinterFilter.try_minterFilterType();
+  if (!minterTypeResult.reverted) {
+    minterFilter.type = minterTypeResult.value;
+  } else {
+    minterFilter.type = "MinterFilterV0";
```

Comment:
This can actually happen on a MinterFilterV1, specifically for ones deployed before 2023, including our Flagship MinterFilterV1 here 😓: https://etherscan.io/address/0x092B8F64e713d66b38522978BCf4649db14b931E#readContract  

Even more frustrating, MinterFilterV0 and V1 have the same read functions!
There might be hope though - if we can read the revert reason, `alertAsCanonicalMinterFilter` only exists on V0 and would revert with "Only Core whitelisted" , whereas on V1 it would revert because the function wouldn't be found..